mycroft/skills/phillips_hue/test/intent/sample6.intent.json

```diff
@@ -0,0 +1,7 @@
+{
+  "utterance": "increase the color temperature of my lights",
+  "intent_type": "IncreaseColorTemperatureIntent",
+  "intent": {
+    "IncreaseColorTemperatureKeyword": "increase the color temperature of my lights"
+  }
+}
```

Comment:
Please, add two intent tests for each intent to provide more coverage for your vocab.